client/less/classic-modal.less

```diff
@@ -0,0 +1,45 @@
+.success {
```

Comment:
I have stripped out as much as I can. These styles are here to bring the modal near to the centre of the screen, and because I cannot put a custom class on the close button.